LG-8809: Improve a11y of "Learn more" links in IdV

matthinz opened the pull request:
## 🎫 Ticket

[LG-8809](https://cm-jira.usa.gov/browse/LG-8809)

## 🛠 Summary of changes

Update privacy-related "Learn more" links in IdV to be more accessible, expanding them out to a full sentence instead of just the text "Learn More".

<!--
## 📜 Testing Plan

Provide a checklist of steps to confirm the changes.

- [ ] Step 1
- [ ] Step 2
- [ ] Step 3
-->

## 👀 Screenshots

If relevant, include a screenshot or screen capture of the changes.

<details>
<summary>English</summary>

<table>

<tr>
<th>Page</th>
<th>Screenshot</th>
</tr>

<tr>
<td>Welcome</td>
<td>

![welcome-en](https://user-images.githubusercontent.com/364697/218575454-050b82cd-effe-4757-88ef-348a503de590.png)


</td>
</tr>

<tr>
<td>Getting started</td>
<td>

![getting-started-en](https://user-images.githubusercontent.com/364697/218575489-21dbb6ab-76ca-4f41-9570-30ae2ec111c6.png)


</td>
</tr>

<tr>
<td>SSN</td>
<td>

![ssn-en](https://user-images.githubusercontent.com/364697/218575529-693b7b70-2227-4dcb-824a-c5cbef911c82.png)


</td>
</tr>

</table>

</details>

<details>
<summary>Spanish</summary>


<table>

<tr>
<th>Page</th>
<th>Screenshot</th>
</tr>

<tr>
<td>Welcome</td>
<td>

![welcome-es](https://user-images.githubusercontent.com/364697/218575562-9a980621-9474-4f71-ba66-af31cdee45fb.png)

</td>
</tr>

<tr>
<td>Getting started</td>
<td>

![getting-started-es](https://user-images.githubusercontent.com/364697/218575688-2cfa27ec-17ff-4585-ad60-19b0f6e568f5.png)


</td>
</tr>

<tr>
<td>SSN</td>
<td>

![ssn-es](https://user-images.githubusercontent.com/364697/218575754-d50ed463-0f2d-4a00-9cc6-e6826865abc3.png)


</td>
</tr>

</table>

</details>

<details>
<summary>French</summary>


<table>

<tr>
<th>Page</th>
<th>Screenshot</th>
</tr>

<tr>
<td>Welcome</td>
<td>

![welcome-fr](https://user-images.githubusercontent.com/364697/218575790-8e927bb9-fa56-4891-93f2-1a6814247872.png)

</td>
</tr>

<tr>
<td>Getting started</td>
<td>

![getting-started-fr](https://user-images.githubusercontent.com/364697/218575842-500fd243-2663-4a18-a24c-4494d2ba0751.png)


</td>
</tr>

<tr>
<td>SSN</td>
<td>

![ssn-fr](https://user-images.githubusercontent.com/364697/218575871-e9ec6a70-4760-4a9c-8a16-673e4fd9804a.png)


</td>
</tr>

</table>
</details>

artfulaction commented:
LGTM, I'm thinking just for all 3 screens, we can add a <br> between the body and the link to add some visual separation. Thank you! :)

benjaminchait commented:
> Question based on the screenshots: Are we okay that the link breaks onto its own line, vs. staying inline with the paragraph text? Not sure how technically possible this is to change, with the link being a separate element with its own box styling.

I'm curious, what's causing the initial line break? (Is this what @aduth means by "the link being a separate element with its own box styling"? Is this a CSS box model thing?) 

cc @matthinz

matthinz commented:
@benjaminchait Looks like we are using `display: inline-block` on offsite links, which forces the whole element onto a new line if it overflows the line it is on.

matthinz commented:
@artfulaction (and others on this chain) I've pushed an update (and updated the screenshots above) to add more vertical spacing for the link.

Under the hood, this meant moving the link out into its own `<p>` tag. 

In the course of this I read up on `<label>` and found [accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#interactive_content) with including `<a>` elements inside `<label>` tags, so I updated the "How verifying your identity works" screen to move the link outside of the checkbox label.

artfulaction reviewed (approved):
LGTM! Thanks for the explanation and for being thoughtful about the accessibility concerns!
